applications/DG-Max/Algorithms/DGMaxDiscretization.cpp

```diff
@@ -805,12 +805,20 @@ std::map<typename DGMaxDiscretizationBase::NormType, double>
         for (typename Base::MeshManipulator<DIM>::FaceIterator it =
                  mesh.faceColBegin();
              it != end; ++it) {
+            if (!(*it)->isOwnedByCurrentProcessor()) {
```

Comment:
With this you only work on what is owned by the current processor. Before it was operating on everything.